jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/ConfigMapEnricherTest.java

```diff
@@ -43,9 +43,9 @@
     private JKubeEnricherContext context;
 
     @Test
-    public void should_materialize_file_content_from_annotation() throws Exception {
+    public void should_materialize_file_content_from_deprecated_annotation() throws Exception {
         final ConfigMap baseConfigMap = createAnnotationConfigMap("test-application.properties",
-                "src/test/resources/test-application.properties");
+                "src/test/resources/test-application.properties", ConfigMapEnricher.PREFIX_ANNOTATION);
```

Comment:
Please, don't use the constant. Use the String literal instead.

---> If the constant is wrong, or changed, the test won't ever fail.